src/java.base/share/conf/security/java.security

```diff
@@ -28,6 +28,33 @@
 # Properties in this file are typically parsed only once. If any of the
 # properties are modified, applications should be restarted to ensure the
 # changes are properly reflected.
+#
+# The special "include" property can be defined one or multiple times in
+# this file with a filesystem path value. The effect of each definition
```

Comment:
"this file". Is this precise? As described below, It can also appear in included files and the file "java.security.properties" points to.

Maybe not an issue since no one will be confused. You decide whether to make any update.

Reply:
I don't see "this file" as strictly necessary given the context so we can remove it.

Reply:
Done in 580d34a63b5d3b6c7d2323413f338527db2d9acd.